tests/framework/db/QueryBuilderTest.php

```diff
@@ -2523,6 +2523,17 @@ public function testBuildLikeCondition($condition, $expected, $expectedParams)
         $this->assertEquals($expectedParams, $params);
     }
 
+    /**
+     * @see https://github.com/yiisoft/yii2/issues/18134
+     */
+    public function testExpressionIsNotQuotedInColumnName()
```

Comment:
I would move this test to `likeConditionProvider()`.

Reply:
moved